build.sh

```diff
@@ -56,7 +54,8 @@ esac
 
 export INSTALL
 
-$MAKE -e
+$MAKE 
 $MAKE -C `pwd`/libsrc
 $MAKE -C `pwd`/libsrc install
 $MAKE -C `pwd`/libsrc/_DEVELOPMENT
+$MAKE -C test
```

Comment:
Ah yes, I noticed that the tests are now run as part of the build. I will probably separate them for porting, but I think it's fine to run the tests like this for the main repo.